### ISSUE TITLE
Dependencies outdated

### ISSUE THREAD

**Issue by johnjeremih** [udacity/MyGarden#22]
I created a pull request to update all the dependencies due to the project being so old.

**Comment by johnjeremih** [comment]
?

**Comment by thiagofnunes** [comment]
This is silly.
All the libraries are outdated, no one is doing anything about it.
It feels like they created the Android Udacity course and left it there.

**Comment by johnjeremih** [comment]
@thiagofnunes  you are correct, It was relevant when I did the course. Now is not. I can help you If you want.

**Comment by thiagofnunes** [comment]
It is fine John, thank you so much.
It is just a pain in the ass having to update everything after you download the initial code.
And then I did get some bugs related to the new libraries and all that, but managed to fix them.
Thank you so much for your support though.

**Comment by LaszloLajosT** [comment]
Hi guys,

I completed the Android Basics from Udacity last week and I started to search some code from Android developer program in advance.

I can tell @johnjeremih ,  most of the codes were outdated from the Basics course, I wouldn't be surprised anymore if it is the same all way long in the advance course.
Anyway, I don't think anyone checking these pull requests what we make.  But I really appreciate all the students commitment and their updated starter code. 

 I usually share my starter code with other students when they check the practise solutions along the course.  I just hope, that one day somebody from Udacity will update these two courses for new students.
